components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -169,9 +172,11 @@ public boolean doPostSetUserClaimValueWithID(String s, UserStoreManager userStor
     public boolean doPreSetUserClaimValueWithID(String userID, String claimURI, String claimValue, String profileName,
                                                 UserStoreManager userStoreManager) throws UserStoreException {
 
-        // Validate whether claim update request is for a provisioned user.
-        if (FrameworkUtils.isJITProvisionEnhancedFeatureEnabled()) {
-            validateClaimUpdate(getUsernameFromUserID(userID, userStoreManager));
+        if (FrameworkUtils.isJITProvisionEnhancedFeatureEnabled() && StringUtils.isNotBlank(claimURI)) {
+            // Check whether if this is an account locking scenario.
+            boolean isAccountLockRequest = claimURI.equals(ACCOUNT_LOCK_CLAIM_URI);
```

Comment:
Any possibility of NPE? If so better to use StringUtils.

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/376/commits/01ac2420b372fb1cec7701640adfb172f6681823